README.md

```diff
@@ -144,19 +144,20 @@ steps:
 
 ### Specify the semantic versioning category to use
 
-By default, this GitHub Action assumes merge category is `MAJOR`. This feature
-looks at the title of the PR to figure out what is being updated. A dependabot
-style title such as: `chore(deps): bump @types/jest from 26.0.12 to 27.0.13` is expected. You
-can override the merge category by changing the value of `PRESET` parameter (one
-of `MAJOR`, `MINOR` or `PATCH`):
+By default, this GitHub Action assumes merge category is `DEPENDABOT_MAJOR`.
```

Comment:
Well, maybe by default we should have no presets? I mean, keep the current behavior, merge everything.

Reply:
Having `DEPENDABOT_MAJOR` means that everything will be merged (major, minor and patch updates)

Reply:
Doesn't that suggest the someone should use this action with Dependabot? I'm not sure if that's an intuitive default, I can see it causing confusion.

Reply:
hmm, I c what you mean. I'll change as suggested

Reply:
Done